doc/quickstart/02_passive_node.md

```diff
@@ -62,7 +62,9 @@ Description of the fields:
       - `file`: path to the log file.
 - `rest`: (optional) Configuration of the REST endpoint.
     - `listen`: _address_:_port_ to listen for requests
-    - `pkcs12`: (optional) Certificate file
+    - `tls`: (optional) enables TLS and disables plain HTTP if provided
+      - `server_certs_file`: path to server X.509 certificate chain file, must be PEM-encoded and contain at least 1 item
```

Comment:
Could do away with the `server_` prefix (we don't have a REST client side in jormungandr), and maybe bikeshed it as `cert_file` if it's going to carry just one self-signed certificate most of the time; that will be more consistent with the `*CertFile*` error variants.